nan_maybe_43_inl.h

```diff
@@ -230,7 +230,7 @@ NAN_INLINE MaybeLocal<v8::Object> CloneElementAt(
   }
   return handle_scope.Escape(obj->Clone());
 #else
-  return array->CloneElementAt(context, index);
+  return array->CloneElementAt(GetCurrentContext(), index);
```

Comment:
I think the handle returned from `GetCurrentContext` now leaks here instead. It would be better to move the handle scope out before the preprocessor block.